build_tools/ci/cpu_comparison/matmul_template/matmul_truncf_MxK_KxN.mlir

```diff
@@ -0,0 +1,13 @@
+// input ${M}x${K}x${TYPE1}
+// input ${K}x${N}x${TYPE1}
+
```

Comment:
You have a template version of the test and a non-template version -- you can remove one of them I think?

Reply:
Have removed the non-template version.